server/tests/article.test.js

```diff
@@ -270,7 +284,21 @@ describe('Article test', () => {
       });
   });
 
-  it('no tagList: should throw an error if tagList is not provided', (done) => {
+  // it('should return 400 if taglist does not follow specified format', (done) => {
```

Comment:
This should be deleted if it's not necessary.

Reply:
Yea.

Reply:
Thanks for the observation will do that.